r/R/rzeppelin/R/common.R

```diff
@@ -0,0 +1,14 @@
+strintrplt <- function(snippet,envir=parent.frame()) {
```

Comment:
Makes perfect sense, thank you for explanation. 
I believe that is the case where changed one is distributed under the same licence, as per [#4 in licence doc](http://www.apache.org/legal/src-headers.html#3party).